src/utils.js

```diff
@@ -70,6 +70,26 @@ const filterLargeUrl = (url, urlLengthLimit) => {
   return url.length > urlLengthLimit ? truncateUrl(url) : url
 }
 
+const parsePropsType = (reqProps, propsType) => {
+  const propsTypeKeys = R.keys(propsType)
+
+  if (propsTypeKeys.length === 0) {
+    return reqProps
+  }
+
+  propsTypeKeys.forEach(key => {
+    const propPath = key.split('.')
+    const reqProp = R.path(propPath, reqProps)
+
+    if (reqProp) {
```

Comment:
Se o valor for `0` ou `false`, não vai ser logado. 
```suggestion
    if (reqProp !== undefined) {
```

Reply:
I don't think that's necessary since if the value were either  `0` or `false`  it would break at runtime, both `request-logger` and `response-logger`  depends on `url` and `body` properties in order to work

Reply:
What if a request have `body.amount` as a number, and I want to log it as a string?
If the `body.amount` is the number `0`, it wouldn't be logged.

Reply:
oops,  I was referring to the other if statement above :stuck_out_tongue: , my bad 

That indeed would happen, thanks for pointing that out!